00-RELEASENOTES

```diff
@@ -1,16 +1,284 @@
+
+Behavior Changes - Core
+=======================
+* Re-brand the Lua debugger. (#603)
+* Change default pidfile from `redis.pid` to `valkey.pid`. (#378)
+* Abort transactions on nested `MULTI` or `WATCH` commands. (#723)
+* Ensure keys that match the `SCAN` filter are not lazily expired and return
+  an error for invalid types. (#501)
+* Rename `redis` in AOF logs and proc title to `valkey-aof-rewrite`. (#393)
+* Change default syslog-ident from `redis` to `valkey`. (#390)
+* Update `Redis` to `Valkey` in `serverLog` messages in server.c file. (#231)
+* Remove `Redis` from various error reply messages. See GitHub PR for more
+  details. (#206)
+* Reject empty strings for configs `dir`, `dbfilename`, and `cluster-config-file`.
+  (#636)
+* Change key-spec flag from `RW` to `OW` for `SINTERSTORE` command. (Redis#12917)
+* Return more precise error messages for some cases verifying keys during script
+  execution. (Redis#12707)
+* Return errors for `BITCOUNT` and `BITPOS` with non-existing keys or invalid
+  arguments instead of zero. (Redis#11734)
+* Validate `BITCOUNT` arguments before key existence check. (Redis#12394)
+* Redact ACL username information and mark `*-key-file-pass` configs as
+  sensitive. (Redis#12860)
+
+Behavior Changes - Cluster
+==========================
+* Allow `CLUSTER NODES/INFO/MYID/MYSHARDID` during loading state. (#596)
+* Make cluster replicas return `ASK` and `TRYAGAIN` during slot migration. (#495)
+
+Behavior Changes - Sentinel
+===========================
+* Replace `master-reboot-down-after-period` with `primary-reboot-down-after-period`
+  in `sentinel.conf`. (#647)
+
+Bug Fixes - Core
+================
+* Fix a bug that caused LRU/LFU inconsistencies for some integer objects. (#250)
+* Fix a bug where Valkey may use a sub-optimal encoding for some data types.
+  (Redis#13148)
+* Fix propagation of `entries_read` by calling `streamPropagateGroupID`
+  unconditionally. (Redis#12898)
+* Fix race condition issues between the main thread and module threads.
+  (Redis#12817)
+* Wake blocked clients ASAP in next `beforeSleep` for `WAITAOF`. (Redis#12627)
+* Fix crash in crash-report and improve thread management with RW locks.
+  (Redis#12623)
+
+Bug Fixes - Cluster
+===================
+* Fix a bug where a shard returns the incorrect slot slot information in
+  `CLUSTER SHARDS` command on primary failure. (#790)
+* Allow module authentication to succeed when the cluster is down. (#693)
+* Fix `PONG` message processing for primary-ship tracking during fail-overs.
+  (Redis#13055)
+* Prevent double freeing of cluster link with `DEBUG CLUSTERLINK KILL`.
+  (Redis#12930)
+* Unsubscribe all clients from replica for shard channel if the primary
+  ownership changes. (Redis#12577)
+
+Bug Fixes - CLI
+===============
+* Fix `valkey-check-aof` misidentifying data in manifest format as MP-AOF.
+  (Redis#12958)
+* Fix `valkey-cli` to respect the `--count` option without requiring
+  `--pattern`. (Redis#13092)
```

Comment:
Another one I missed, #12986 "Pick random slot for a node to distribute operation across slots in redis-benchmark #12986". An update to redis-benchmark so that it would randomly send traffic across slots.